### ISSUE TITLE
deploy_docs.sh is failing with an authentication error

### ISSUE THREAD

**Issue by davidwagner** [data-8/datascience#363]
In the last build where docs successfully built on Travis (https://travis-ci.org/data-8/datascience/builds/402003674#L2137-L2141), uploading the docs failed because of an authentication error:

```
after_success.2
7.73s$ bash tools/deploy_docs.sh
Starting virtual X frame buffer: Xvfb.
[...]
cd doc_build && \
		git add -A && \
		git commit -m "Generated by commit 5af00d417afb2c034c1346bc90d9c51b3fc66b06, pushed by Travis build 924." && \
		git push -f https://[secure]@github.com/dsten/datascience.git gh-pages
[gh-pages bf4d23e] Generated by commit 5af00d417afb2c034c1346bc90d9c51b3fc66b06, pushed by Travis build 924.
 107 files changed, 3323 insertions(+), 2881 deletions(-)
[...]
remote: Invalid username or password.
fatal: Authentication failed for 'https://[secure]@github.com/dsten/datascience.git/'
```

I think the culprit is that the `GH_REPO` environment variable is set incorrectly:

```
$ export GH_REPO=github.com/dsten/datascience.git
```

It looks like this comes from the `GH_REPO:` line in `.travis.yml`.  I think a while ago the dsten organization on Github got renamed to data-8, and this never got updated, so we probably need to edit .travis.yml to update the `GH_REPO` variable.

I suspect we haven't seen this recently because building the docs has been failing for a while, so the script hasn't gotten to the point of trying to push the built docs to Github; once we fix things so docs build, it'll try to push them to Github and likely trigger this error again.

**Comment by adnanhemani** [comment]
Even after updating the references from"dsten" to "data-8", we're having the same [problems](https://travis-ci.org/data-8/datascience/jobs/546302918#L2310)...

**Comment by adnanhemani** [comment]
I'm not sure where the username/pw is on Travis for it to upload to "gh-pages". I checked the Travis config and didn't find anything there either? Does anyone else know where the authentication is coming from? @SamLau95 ?

**Comment by davidwagner** [comment]
`.travis.yml` sets [environment variables](https://docs.travis-ci.com/user/environment-variables/): see the `env.global` section.  The `secure` part is an encrypted variable name + value pair, [encrypted so only Travis can decrypt](https://docs.travis-ci.com/user/encryption-keys/).  It seems to be of the form `GH_TOKEN=secret` where `secret` is replaced with an [authentication token from Github](https://help.github.com/en/articles/git-automation-with-oauth-tokens).  There's where these lines at the top of the log come from:

```
Setting environment variables from .travis.yml
$ export GH_REPO=github.com/data-8/datascience.git
$ export GH_TOKEN=[secure]
```

That said, I don't know why it is still failing.  Perhaps the token is no longer valid?  (If so we'd need to create a new one, encrypt it, and then update the `secure` line in `.travis.yml`.)  Maybe the old token was invalidated by the name change from dsten to data-8 and that made the token no longer valid?  I don't have any other guesses at this point.  If we're stuck, perhaps we could look through old travis logs and find the last point where it still worked and see if that gives us any clues.

**Comment by adnanhemani** [comment]
Closed by #379
